src/Runner.Listener/JobDispatcher.cs

```diff
@@ -55,6 +57,11 @@ public sealed class JobDispatcher : RunnerService, IJobDispatcher
 
         private TaskCompletionSource<bool> _runOnceJobCompleted = new TaskCompletionSource<bool>();
 
+        public event EventHandler<EventArgs> JobStarted;
```

Comment:
What about we only have one `EventHandler<JobStatusEventArgs> JobStatus` and use the `EventArgs` to distinct JobStart vs JobComplete?